### PR TITLE
Readme and assembly module

### PR DESCRIPTION
**Pull request by javelinjs**
* Assembly jar, which can be integrated into other projects.
* Build and usage instructions in README.md

@tqchen @yanqingmen @terrytangyuan 
We are almost set for merge, while there still exists some bugs and tests to fix. I'll raise a PR towards dmlc/mxnet early next week.
And please help me to review this simple readme document. Thank you.

### CODE REVIEW

**Comment by tqchen** [comment]
Great Job! Thanks for all the effort

File: scala-package/README.md [hunk 1]
```diff
@@ -1,5 +1,129 @@
...
+  numInst += 1
+}
+val acc = numCorrect.toFloat / numInst
+println(s"Final accuracy = $acc")
+```
+
+You can refer to [MXNet Scala Package Examples](https://github.com/javelinjs/mxnet-scala-example)
+for more information about how to integrate MXNet Scala Package into your own project.
+Currently you have to put the Jars into your project's build classpath manully.
+We will provide pre-built binary package on [Maven Repository](http://mvnrepository.com) soon.
```
Comment (tqchen) [inline scala-package/README.md]: does pre-build contain the GPU dll?

Comment (javelinjs) [inline scala-package/README.md]: I'm willing to provide both CPU-only and GPU version, for OSX, Linux and Windows(maybe later) platform.

**Comment by terrytangyuan** [comment]
Looks great! Look forward to the merge!

**Comment by tqchen** [comment]
I am not sure how we should host the doc. One way we can do is simply do it togther with readthedocs. The only thing we need to figure out is build of scala-docs.

**Comment by terrytangyuan** [comment]
Maybe a short description for our names in CONTRIBUTORS.md? @tqchen Could you help and just add those into the main repo?

**Comment by javelinjs** [comment]
@tqchen I agree with you that we shall put docs in readthedocs, while it requires a little bit more efforts. I will try to make it in next few weeks.

**Comment by jermainewang** [comment]
These are great job! Totally agree on including them as committer.

**Comment by javelinjs** [comment]
Yeah, thanks for everybody's effort. I'll list the contributions together with the coming PR description.
And if you have time, please also help me to test the example module and https://github.com/javelinjs/mxnet-scala-example on your own environment. Looking forward to your feedback :)

**Comment by yanqingmen** [comment]
Great~ I'll test on my environment~

**Pull request merged**